languages/elixir/config.json

```diff
@@ -8,6 +8,52 @@
     "indent_style": "space",
     "indent_size": 2
   },
+  "tags": [
+    "paradigm/functional",
+    "typing/dynamic",
+    "typing/strong",
+    "execution_mode/compiled",
+    "platform/windows",
+    "platform/mac",
+    "platform/linux",
+    "runtime/beam",
+    "used_for/backends",
+    "used_for/embedded_systems",
+    "used_for/financial_systems",
+    "used_for/web_development"
```

Comment:
"used for" as suggested by people in https://exercism-team.slack.com/archives/CAS9QNR7E/p1611235680002200